### PR TITLE
Translations from 20191210.2

### PR DESCRIPTION
**Pull request by mcumming**
Updating translations from 20191210.2 drop.

### CODE REVIEW

File: Xamarin.PropertyEditing/Properties/Resources.es.resx [hunk 1]
```diff
@@ -795,6 +906,142 @@
...
+  </data>
+  <data name="AccessibilityBindingAddValueConverterCancel" xml:space="preserve">
+    <value>Este botón cancelará los cambios que ha realizado en el editor para agregar convertidor de valores.</value>
+    <comment>Text to inform the user that this button will cancel the changes you have made in the Add Value Converter Editor</comment>
+  </data>
+  <data name="ObjectTypeLabelNone" xml:space="preserve">
+    <value>Ninguno</value>
+  </data>
+  <data name="FilterTypePlaceholder" xml:space="preserve">
+    <value>Filter Types</value>
```
Comment (CartBlanche) [inline Xamarin.PropertyEditing/Properties/Resources.es.resx]: This translation has not been done.

File: Xamarin.PropertyEditing/Properties/Resources.ru.resx [hunk 2]
```diff
@@ -795,6 +906,142 @@
...
+  </data>
+  <data name="AccessibilityBindingAddValueConverterCancel" xml:space="preserve">
+    <value>Эта кнопка позволяет отменить изменения, внесенные в редакторе "Добавление преобразователя значений".</value>
+    <comment>Text to inform the user that this button will cancel the changes you have made in the Add Value Converter Editor</comment>
+  </data>
+  <data name="ObjectTypeLabelNone" xml:space="preserve">
+    <value>Нет</value>
+  </data>
+  <data name="FilterTypePlaceholder" xml:space="preserve">
+    <value>Filter Types</value>
```
Comment (CartBlanche) [inline Xamarin.PropertyEditing/Properties/Resources.ru.resx]: This translation has not been done.

File: Xamarin.PropertyEditing/Properties/Resources.es.resx [hunk 3]
```diff
@@ -511,76 +574,91 @@
...
+    <comment>Text to denote that an the user would like to select an item</comment>
   </data>
   <data name="SelectTypeForProperty" xml:space="preserve">
-    <value>Seleccionar tipo de {0}</value>
-    <comment>Select {property} type</comment>
+    <value>Este botón es para la propiedad {0}.</value>
+    <comment>Select {Property} type</comment>
   </data>
   <data name="PropertyFilterLabel" xml:space="preserve">
     <value>Property Filter</value>
```
Comment (CartBlanche) [inline Xamarin.PropertyEditing/Properties/Resources.es.resx]: This property still hasn't been done.

File: Xamarin.PropertyEditing/Properties/Resources.ru.resx [hunk 4]
```diff
@@ -511,76 +574,91 @@
...
+    <comment>Text to denote that an the user would like to select an item</comment>
   </data>
   <data name="SelectTypeForProperty" xml:space="preserve">
-    <value>Выберите тип {0}</value>
-    <comment>Select {property} type</comment>
+    <value>Это кнопка для свойства {0}.</value>
+    <comment>Select {Property} type</comment>
   </data>
   <data name="PropertyFilterLabel" xml:space="preserve">
     <value>Property Filter</value>
```
Comment (CartBlanche) [inline Xamarin.PropertyEditing/Properties/Resources.ru.resx]: This property still hasn't been done.

**Comment by garuma** [comment]
@CartBlanche yep, it's a little strange but that's how the translation pipeline works, their tooling will first ensure that every string is available in the language specific file (with its English version to start) to avoid missing resource entries. The translator will then go and progressively change those strings to the real version. This allow us to do incremental drop like this and it will get better each time around.

**Comment by CartBlanche** [comment]
@garuma But at least one of the translation resources has been there for months, if not years and hasn't been done, so we need to be able to highlight that certain ones need prioritising. Like the the 2 I pointed out above, which are related to the DevDiv issue we spoke about yesterday.

**Pull request merged**